project_stock/migrations/13.0.1.2.0/post-migration.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    env.cr.execute(
```

Comment:
There's no need of SQL here. You can do a simply search over tasks with the o2m field.